### PR TITLE
Create a new contact UI

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by vijay2136**
fixes #5 
- Created new contact ui
- Fixed feedback comments

### CODE REVIEW

File: app/javascript/src/components/Common/Header.jsx [hunk 1]
```diff
@@ -3,13 +3,13 @@ import React, { useState } from "react";
...
 
-const Header = ({ title, setShowMenu, setShowPane }) => {
+const Header = ({ title, setShowMenu, setShowPane, showMenu }) => {
   const [searchTerm, setSearchTerm] = useState("");
 
   return (
     <NeetoHeader
       title={`All ${title}`}
-      menuBarToggle={() => setShowMenu(prev => !prev)}
+      menuBarToggle={() => setShowMenu(!showMenu)}
```
Comment (yedhink) [inline app/javascript/src/components/Common/Header.jsx:12]: Not sure why this change was made over here. What's your strong reasoning for doing this?

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import formValidationSchemas from "constants/formValidationSchemas";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:7]: What's the point of keeping validation schemas in `constants` folder? I had already given many review for you pointing out to keep constants within the closest scope.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  return (
+    <Formik
+      initialValues={contact}
+      onSubmit={handleSubmit}
+      validationSchema={formValidationSchemas.contactsForm}
+    >
+      {({ isSubmitting }) => (
+        <FormikForm className="w-full">
+          <Pane.Body>
+            <div className="space-y-8 w-full">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:21]: Is there a need for this `div`? What's the issue with passing the className along with the `Pane.Body`?

**Review by yedhink — COMMENTED** [review]
@vijay2136 

Few comments.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 3 above)
Comment (vijay2136) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:21]: This div here is used to give "w-full" property to all of the child components. Otherwise this class property should be added separately to all individual components.

File: app/javascript/src/components/Common/Header.jsx (hunk 1 above)
Comment (vijay2136) [reply app/javascript/src/components/Common/Header.jsx:12]: It was a mistake from my side. Resolved in #14

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 3 above)
Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:21]: @vijay2136 The question was on why we are not able to pass this same `className` attr to `Pane.Body`(which is the parent for all the children) and avoid having this `div`.

Comment (vijay2136) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:21]: https://vimeo.com/670721598/64d9e2df9d

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:21]: @vijay2136 

You still haven't addressed this part:

> The question was on why we are not able to pass this same className attr to Pane.Body

```jsx
<Pane.Body className="space-y-8 w-full">
{children without any wrapping div as parent}
</Pane.Body>
```

Please make a video and show me how the UI behaves in this case.
